code/modules/mob/living/simple_animal/hostile/megafauna/hivetechboss.dm

```diff
@@ -11,12 +11,12 @@
 	pixel_x = -16
 	ranged = TRUE
 
-	health = 2500
-	maxHealth = 2500
+	health = 2100
+	maxHealth = 2100 //Only way for it to show up right now is via adminbus OR Champion call (which gives it 150hp). For comparison Kaiser has 2000hp
 	break_stuff_probability = 95
 
-	melee_damage_lower = 10
-	melee_damage_upper = 20
+	melee_damage_lower = 40
+	melee_damage_upper = 50 //don't try going into melee with this unless you are pumped full of brute healing chems, seriously.
```

Comment:
That's really too much. Keep it at 35 max.

Reply:
maybe 35 lower and 40 upper? There's only one way for it to show up, will show up only once, **when it dies the entire hive dies along with it** and it's a 1 in a 5 chance.

Reply:
Then it's a reason to increase HP, not damage. Almost one shot mobs are not fun to fight against.

Reply:
And yeah, contributing guidelines advises really hard against buffing or nerfing something too much in one go.

Reply:
alright, will reduce the damage to 35 upper, 30 lower and increase the HP by 150 if that's fine